doc/fparser.rst

```diff
@@ -137,7 +137,8 @@ that reader to create the tree, e.g.:
 
 ::
 
-  >>> reader = FortranStringReader(code, False, True)
+  >>> reader = FortranStringReader(code)
```

Comment:
Nicely done. Since you made changes in this file, could you please update the "Modified work Copyright 2017... " line at the top to 2017-2018?

Reply:
Well spotted - done.